.github/workflows/code_coverage.yml

```diff
@@ -27,16 +27,6 @@ jobs:
         uses: Swatinem/rust-cache@v2.2.1
       - name: Install grcov
         run: if [[ ! -e ~/.cargo/bin/grcov ]]; then cargo install grcov; fi
-      - name: Build simulator image
```

Comment:
Can we just comment this one instead of deleting it? With a `TODO: re-enable the hwi tests`

Reply:
Done! Glad that you are working on re-enabling that.